### ISSUE TITLE
Update click shipping task logic

### ISSUE THREAD

**Issue by chihsuan** [woothemes/woocommerce#33364]
**User story:**
As a new merchant
I want to focus on the most important setup tasks
So I can review some details later

**Description:**
With Jetpack installed and connected, alongside WCS, we have the opportunity to set things up on the users' behalf, and ask them to review and confirm, rather than start from scratch and think.

**Design overview (with new spotlight component):**
cJQeyyhx9PraQrY3YAtCGM-fi-705%3A80783

**Acceptance Criteria:**

~Case 1:~ There is a separate ticket for this case https://github.com/woocommerce/woocommerce/issues/33526

- If the store sells physical products, has JP and WCS installed and connected, and is located in the US
- If the store sells physical products, and is not located in US/Canada/Australia/UK (irrelevant if JP is installed or not):
- If the store sells physical products and is located in US, but JP and WCS are not installed:

- [ ] Address users to WooCommerce > Settings > Shipping once clicked on the task on the home page

**Case 2:**
-  If we don't know if the store sells physical products and where it's located, and JP is **not** installed:

- [ ] Address users to a single page with the following steps: Set your store location, Review your shipping options, Enable shipping label printing and discounted rates (only if the store location added in the first step is US, Canada or Australia), **Connect your store** 

**Case 3:**
-  If we don't know if the store sells physical products and where it's located, and JP is installed:

- [ ] Address users to a single page with the following steps: Set your store location, Review your shipping options, Enable shipping label printing and discounted rates (only if the store location added in the first step is US, Canada or Australia) 

**Case 4:**
- If the store is located in the UK, Australia or Canada

- [ ]  Same flow as now with a few optimizations in terms of copy and visual. (https://github.com/woocommerce/woocommerce/issues/33368)

ref: https://github.com/woocommerce/woocommerce/issues/32318

**Gherkin syntax**

~Case 1~ There is a separate ticket for this case https://github.com/woocommerce/woocommerce/issues/33526

**Case 2**
Given a store with the following OBW selections and Jetpack is not installed

| Type | Value |
|------|-------|
| Product type | unknown| 
| Store Details[Country / Region] | unknown|

When a user clicks on the shipping task on WooCommerce -> Home
Then redirect the user to the shipping task with the following additional steps

- Set your location
- Review your shipping options
- Enable shipping label printing and discounted rates
- Connect your store

**Case 3**

Given a store with the following OBW selections and Jetpack is installed

| Type | Value |
|------|-------|
| Product type | unknown| 
| Store Details[Country / Region] | unknown|

When a user clicks on the shipping task on WooCommerce -> Home
Then redirect the user to the shipping task with the following additional steps

- Set your location
- Review your shipping options
- Enable shipping label printing and discounted rates

**Case 4**
Given a store located in the UK or Australia or Canada
When a user clicks on the shipping task on WooCommerce -> Home
Then use the same flow as now with a few optimizations in terms of copy and visual.

**Comment by moon0326** [comment]
Hey @verofasulo  ✋ 

Sorry for the late ping. I should have checked this before starting the sprint work.

I noticed that we don't actually display the shipping task if a user skips the OBW. 

Do we want to change the logic to display the shipping task all the time and add the store details fields (address, country, state, etc) on the shipping task?

**Comment by moon0326** [comment]
Hey @verofasulo ✋

Sorry for the late ping. I should have checked this before starting the sprint work.

I noticed that we don't actually display the shipping task if a user skips the OBW.

This is the list of tasks when OBW is skipped.
![Screen Shot 2022-06-21 at 6 24 04 PM](https://user-images.githubusercontent.com/4723145/174923449-0543a949-70f7-4b61-a51a-e1acb4f9fecc.jpg)


Do we want to change the logic to display the shipping task all the time and add the store details fields (address, country, state, etc) on the shipping task?

**Comment by chihsuan** [comment]
I think we'll change the logic to display the shipping task in https://github.com/woocommerce/woocommerce/issues/33363.

In this case, if a user skips OBW, we consider as they sell physical products but we don't know where it's located.

>  display the "Set up shipping" task in the "Get ready to start selling" section when either of the following is true:
> - **We don't know** if the store sells physical products and **where it's located**, and JP is not installed

**Comment by moon0326** [comment]
> I think we'll change the logic to display the shipping task in #33363.
> 
> In this case, if a user skips OBW, we consider as they sell physical products but we don't know where it's located.
> 
> > display the "Set up shipping" task in the "Get ready to start selling" section when either of the following is true:
> > 
> > * **We don't know** if the store sells physical products and **where it's located**, and JP is not installed

Thank you!

**Comment by verofasulo** [comment]
Hi @moon0326 ,

Yes, as @chihsuan said, if the user skips the profiler, we display the shipping task. :)

**Comment by moon0326** [comment]
Hey @verofasulo ✋ 

I have a question about the `Set store location` fields.

From the Figma, we have the following design

![Screen Shot 2022-06-23 at 8 40 32 PM](https://user-images.githubusercontent.com/4723145/175457562-0dc4dec8-6f1c-4e87-9e18-80e1c79b0e27.jpg)

and this is what we currently have

![Screen Shot 2022-06-23 at 8 29 01 PM](https://user-images.githubusercontent.com/4723145/175457589-77ec71fe-71d5-4f58-8671-01fb253601a6.jpg)

What do you think about keeping the current design (form fields, not the extra text copy) as it is since the same design is used in the OBW section? I figured it would be nice to keep the consistent look

**OBW store details**
![Screen Shot 2022-06-23 at 8 29 10 PM](https://user-images.githubusercontent.com/4723145/175457681-5ef37380-41f2-4650-b24b-de3db61fd5ce.jpg)

cc @ilyasfoo  since it might apply to you as well.

**Comment by verofasulo** [comment]
Hi @moon0326 !

Good point! The fields order was redesigned thinking about a general overhaul - But it makes sense to align with what we have now and stay consistent with the current order :)
